src/components/Onboarding/Onboarding.tsx

```diff
@@ -1,75 +1,104 @@
 import React, { useContext, useEffect, useState } from 'react';
+import { API, Auth } from 'aws-amplify';
+import { createStackNavigator } from '@react-navigation/stack';
 import { AppContext } from '../../contexts/AppContext';
+import { homeInfo } from './onboardingData';
 import Page1 from './OnboardingPage1';
 import Page2 from './OnboardingPage2';
 import Page3 from './OnboardingPage3';
 import Page4 from './OnboardingPage4';
+import {
+  createHome,
+  createHomeOwner,
+  updateHome,
+} from '../../graphql/mutations';
+import { CreateHomeInput } from '../../API';
+import { getUser } from '../../graphql/queries';
+import { UserContext } from '../../contexts/UserContext';
 
-const Onboarding: React.FC = () => {
-  const [page, setPage] = useState('page1');
-  const [homeData, setHomeData] = useState({
-    AC: '',
-    EV: '',
-    address: '',
-    bathrooms: '',
-    bedrooms: '',
-    city: '',
-    electricEnergyUsage: '',
-    fuelType: '',
-    gasUsage: '',
-    heaterAge: '',
-    hotTub: '',
-    outdoor: '',
-    pool: '',
-    squareFeet: '',
-    state: '',
-    waterFuelType: '',
-    waterHeaterAge: '',
-    waterUse: '',
-    year: '',
-    zipCode: '',
-  });
+const Stack = createStackNavigator();
+
+interface Props {
+  homeInformation?: CreateHomeInput;
+}
+
+const Onboarding: React.FC<Props> = ({ homeInformation }) => {
+  const [page, setPage] = useState('');
+  const [homeData, setHomeData] = useState(
+    homeInformation !== undefined ? homeInformation : homeInfo,
+  );
```

Comment:
nit: TypeScript has something called [nullish coalescing](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing), which lets us simplify ternary checks like this to something like:
```ts
const [homeData, setHomeData] = useState(homeInformation ?? homeInfo);
```
Aside from providing cleaner syntax, this is also beneficial since it checks if `homeInformation === null` as well, which might be useful if somehow `homeInformation` ends up being null (though I'm not sure it would ever be since we're requiring entry in all text fields now, but the idea still holds)

Reply:
^ wasn't aware, def on the side of using that.

Reply:
That's a cool feature I didn't know about! I'll implement this in the next commit.